{{cookiecutter.package_slug}}/tox.ini

```diff
@@ -0,0 +1,59 @@
+[tox]
+env_list =
```

Comment:
I have a limited experience with tox (mostly just testing with multiple python versions). 

Am I correct to assume that it will create a separate, isolated python environment for each of this tools? (e.g. it will take 3 different python interpreters just to run isort black and flake)

Reply:
Yes, I was torn between two approaches: 

1.  Presented here the advantage is that we will have better summary in my opinion but yes it will require to create isolated python environment
2.  Wrapped black isort etc into 'format' step

Generally I am open for both versions.

Reply:
Feels a bit... heavy. It would be nice, if those steps could be merged. What's the difference between output summary in those approaches?